i6_models/parts/factored_hybrid/util.py

```diff
@@ -0,0 +1,63 @@
+from enum import Enum
+from typing import Callable, Union
+
+from torch import nn
+
+
+class PhonemeStateClassV1(Enum):
```

Comment:
```suggestion
class PhonemeBoundaryClassV1(Enum):
```
"state" to me sounds like it would be related to `num_hmm_states_per_phone`

Reply:
>  it would be related to num_hmm_states_per_phone

Yes, I also agree.